src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res

```diff
@@ -36,7 +36,20 @@ let emailField = FormRenderer.makeFieldInfo(
   ~name="email",
   ~placeholder="Enter your Email",
   ~isRequired=false,
-  ~customInput=InputFields.textInput(~autoComplete="off", ()),
+  ~customInput=(~input, ~placeholder as _) =>
+    InputFields.textInput(
+      ~input={
+        ...input,
+        onChange: event =>
+          ReactEvent.Form.target(event)["value"]
+          ->String.trim
+          ->Identity.stringToFormReactEvent
+          ->input.onChange,
+      },
+      ~placeholder="Enter your Email",
+      ~autoComplete="off",
+      (),
+    ),
```

Comment:
try adding test cases for this

Reply:
@JeevaRamu0104 added the test case for same